pages/common/qemu-img.md

```diff
@@ -0,0 +1,19 @@
+# qemu-img
+
+> Tool for QEMU Virtual HDD image creation and manipulation
```

Comment:
I'd use the full name here ("Quick Emulator") since the acronym qemu is already present in the title.